### ISSUE TITLE
Mostly Sunny Icon Missing?

### ISSUE THREAD

**Issue by quadhammer** [DavidFW1960/bom-weather-card#42]
"\\192.168.x.x\config\www\icons\bom_icons\mostly_sunny.png" exists, but it's not being displayed for some reason.

https://imgur.com/a/WrHnz18

Is it just me?

Cheers

**Comment by DavidFW1960** [comment]
I think your path is wrong..
"\192.168.x.x\config\www\icons\weather_icons\animated\mostly_sunny.png"
![image](https://user-images.githubusercontent.com/36390762/100666837-c7d64800-33ad-11eb-8f77-c2029da66856.png)
The one you are referencing is used by the basic card.

mostly_sunny condition uses the fair-day or night icon

you do seem to be conflating the basic card with the full card in your screenshot.

**Comment by quadhammer** [comment]
The full card is working fine, it's the basic card that seem to be missing the icon. I only posted the full card to display what temperature condition was missing, i.e. mostly_sunny

I have:
"\\192.168.x.x\config\www\icons\weather_icons\static\fair-day.svg"
"\\192.168.x.x\config\www\icons\weather_icons\animated\fair-day.svg"
and they work for the full card.

All the others work for the basic card, just not that 'mostly sunny' one for some reason.

**Comment by DavidFW1960** [comment]
He ok!
I can't see any reason why this should not work...
With the forecast card, it should use mostly_sunny.png and I don't see a reason why that would fail unless the icon is missing.
I did add a bunch of new icons when the new component came out as a the new component uses underscores in the condition instead of - signs.

I'll take a fuller look when I'm at my PC tomorrow but I see no reason for this to fail.

**Comment by DavidFW1960** [comment]
Ok so on the mini-card with the 7 day forecasts, mostly_sunny.png should be used. I do see I have a night version of this as well so I can include that in the template for bom_forecast_0

I just dummied this u as a test in the template editor and it looks ok to me
![image](https://user-images.githubusercontent.com/36390762/100805703-46e98000-3483-11eb-836f-646f66e36052.png)

Grab the latest template.md/weather.yaml

Again I don't see any reason why this would not have worked and I had not seen any issues with it. The only change I made is now it will 'see' if it is night for that mostly_sunny and use a night icon.
